Bottom: 3e838b6e6c35b45bbbd72c62fbeb4029be4650df
Top:    e6df0dc7506cc858badc9cb8b9bf916e853521a5
Author: scott snyder <sss@karma>
Date:   2020-02-01 10:42:02 -0500

unique_ptr warning


---

diff --git a/libstdc++-v3/include/bits/unique_ptr.h b/libstdc++-v3/include/bits/unique_ptr.h
index d03266c1878..e738350c3e7 100644
--- a/libstdc++-v3/include/bits/unique_ptr.h
+++ b/libstdc++-v3/include/bits/unique_ptr.h
@@ -461,8 +461,8 @@ _GLIBCXX_BEGIN_NAMESPACE_VERSION
       }
 
       // Disable copy from lvalue.
-      unique_ptr(const unique_ptr&) = delete;
-      unique_ptr& operator=(const unique_ptr&) = delete;
+      unique_ptr(const unique_ptr&) = delete; // If you see an error here, you probably forgot to use std::move with unique_ptr
+      unique_ptr& operator=(const unique_ptr&) = delete; // If you see an error here, you probably forgot to use std::move with unique_ptr
   };
 
   /// 20.7.1.3 unique_ptr for array objects with a runtime length
